<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleDialog</class>
 <widget class="QDialog" name="ScheduleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ì¼ì • ê´€ë¦¬</string>
  </property>
  <property name="styleSheet">
   <string>QDialog {
    background-color: white;
}

QLabel {
    font-size: 13px;
    font-weight: bold;
    color: #333;
}

QLineEdit, QTextEdit, QDateEdit, QTimeEdit, QComboBox {
    padding: 8px;
    border: 2px solid #e0e0e0;
    border-radius: 4px;
    background-color: white;
    font-size: 13px;
}

QLineEdit:focus, QTextEdit:focus, QDateEdit:focus, QTimeEdit:focus, QComboBox:focus {
    border: 2px solid #2196F3;
}

QComboBox::drop-down {
    border: none;
    width: 30px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
    border-top: 5px solid #666;
    margin-right: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string> ì¼ì • ì œëª© *</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="titleEdit">
     <property name="placeholderText">
      <string>ì¼ì • ì œëª©ì„ ì…ë ¥í•˜ì„¸ìš”</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dateLabel">
     <property name="text">
      <string> ë‚ ì§œ *</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dateEdit">
     <property name="maximumDate">
      <date>
       <year>2099</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="timeLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="startTimeLayout">
       <item>
        <widget class="QLabel" name="startTimeLabel">
         <property name="text">
          <string> ì‹œì‘ ì‹œê°„ *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="startTimeEdit">
         <property name="displayFormat">
          <string>HH:mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="endTimeLayout">
       <item>
        <widget class="QLabel" name="endTimeLabel">
         <property name="text">
          <string> ì¢…ë£Œ ì‹œê°„ *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="endTimeEdit">
         <property name="displayFormat">
          <string>HH:mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="categoryLabel">
     <property name="text">
      <string> ì¹´í…Œê³ ë¦¬</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="categoryCombo">
     <item>
      <property name="text">
       <string>ğŸ“‹ ê¸°íƒ€</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ğŸ¢ ì—…ë¬´</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ğŸ‘¥ íšŒì˜</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ğŸ‘¤ ê°œì¸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ğŸ“ í•™ìŠµ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ğŸ¯ í”„ë¡œì íŠ¸</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ğŸ“ ì•½ì†</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="locationLabel">
     <property name="text">
      <string> ì¥ì†Œ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="locationEdit">
     <property name="placeholderText">
      <string>ì¥ì†Œë¥¼ ì…ë ¥í•˜ì„¸ìš” (ì„ íƒì‚¬í•­)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="memoLabel">
     <property name="text">
      <string> ë©”ëª¨</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="memoEdit">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>ë©”ëª¨ë¥¼ ì…ë ¥í•˜ì„¸ìš” (ì„ íƒì‚¬í•­)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #757575;
    color: white;
    border: none;
    padding: 10px 20px;
    border-radius: 6px;
    font-weight: bold;
    font-size: 13px;
}
QPushButton:hover {
    background-color: #616161;
}
QPushButton:pressed {
    background-color: #424242;
}</string>
       </property>
       <property name="text">
        <string>ì·¨ì†Œ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #2196F3;
    color: white;
    border: none;
    padding: 10px 20px;
    border-radius: 6px;
    font-weight: bold;
    font-size: 13px;
}
QPushButton:hover:enabled {
    background-color: #1976D2;
}
QPushButton:pressed:enabled {
    background-color: #1565C0;
}
QPushButton:disabled {
    background-color: #BDBDBD;
    color: #757575;
}</string>
       </property>
       <property name="text">
        <string>ì €ì¥</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
